cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,59 @@
 /// <reference types='cypress' />
 
+const { generateUser } = require('../support/commands.js');
+
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('/automation-practice-form');
   });
 
-  it('', () => {
+  it('should fill all fields in forms except "picture".', () => {
+    const { firstName, lastName, email, phoneNumber, address } = generateUser();
 
+    cy.get('#firstName')
+      .type(firstName);
+    cy.get('#lastName')
+      .type(lastName);
+    cy.get('#userEmail')
+      .type(email);
+    cy.contains('.custom-control-label', 'Male')
+      .click();
+    cy.get('#userNumber')
+      .type(phoneNumber);
+    cy.get('#dateOfBirthInput')
+      .type('{selectAll}09 Jul 2001{enter}');
```

Comment:
Hardcoding the date '09 Jul 2001' can make the tests brittle. Consider generating a random date or getting the current date.